Sparkle/SUAutomaticUpdateAlert.m

```diff
@@ -88,4 +89,30 @@ - (NSString *)descriptionText
     }
 }
 
+- (NSTouchBar *)makeTouchBar
+{
+    NSButton *installButton, *laterButton, *cancelButton;
+    for (NSView *control in self.window.contentView.subviews) {
+        if ([control isKindOfClass:[NSButton class]]) {
+            NSButton *button = (NSButton *)control;
+            if (button.action == @selector(installNow:)) {
+                installButton = button;
```

Comment:
I have a script that copies changes between xibs, so it's enough if you just modify English one.